README.md

```diff
@@ -500,58 +502,54 @@ For Bunyan Stream support, reference our [logdna-bunyan](https://github.com/logd
 
 For Winston support, reference our [logdna-winston](https://github.com/logdna/logdna-winston/) module.
 
-## AWS Lambda Support
+## Using with AWS Lambda
 
-AWS Lambda allows users to add logging statements to their Lambda Functions. You can choose to setup the logger
-as shown above, or you can override the `console.log` and `console.error` statements. AWS Lambda overrides the `console.log`,
-`console.error`, `console.warn`, and `console.info` functions as indicated
-[here](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-logging.html), within the scope of the handler (main)
-function. You can setup an override as follows:
```

Comment:
> AWS Lambda overrides the `console.log`

I don't believe this is true, at least not anymore. In general I've framed this as more of a usage example as there is no built-in "support" needed for Lambda.